Disabling-Upgrades.md

```diff
@@ -0,0 +1,92 @@
+| Fork                   | enderio:hoe                                     |
+| Inventory I            | enderio:inv                                     |
+| Inventory II           | enderio:inv1                                    |
+| Inventory III          | enderio:inv2                                    |
+| Jump                   | enderio:jumpboost1                              |
+| Jump II                | enderio:jumpboost2                              |
+| Jump III               | enderio:jumpboost3                              |
+| Night Vision           | enderio:nightvision                             |
+| Padding                | enderio:padding                                 |
+| Simple Solar           | enderiomachines:solar                           |
+| Solar I                | enderiomachines:solar1                          |
+| Solar II               | enderiomachines:solar2                          |
+| Solar III              | enderiomachines:solar3                          |
+| Sound Locator          | enderio:sounddetector                           |
+| Speed                  | enderio:speedboost1                             |
+| Speed II               | enderio:speedboost2                             |
+| Speed III              | enderio:speedboost3                             |
+| Spoon                  | enderio:spoon                                   |
+| Step Assist            | enderio:step_assist                             |
+| Flippers               | enderio:swim                                    |
+| Explosive I            | enderio:tnt                                     |
+| Explosive II           | enderio:tnt1                                    |
+| Explosive III          | enderio:tnt2                                    |
+| Explosive IV           | enderio:tnt3                                    |
+| Explosive V            | enderio:tnt4                                    |
+| Travel                 | enderio:travel                                  |
+| The One Probe          | enderio:top                                     |
+| Naturalist's Eye       | enderiointegrationforestry:naturalist_eye       |
+| Apiarist's Hat         | enderiointegrationforestry:apiarist_armor_head  |
+| Apiarist's Shirt       | enderiointegrationforestry:apiarist_armor_chest |
+| Apiarist's Pants       | enderiointegrationforestry:apiarist_armor_legs  |
+| Apiarist's Shoes       | enderiointegrationforestry:apiarist_armor_feet  |
+| Revealing              | enderio:gogglesofrevealing                      |
+| Thaumaturge's Robe     | enderio:thaumaturge_robe_chest                  |
+| Thaumaturge's Leggings | enderio:thaumaturge_robe_legs                   |
+| Thaumaturge's Boots    | enderio:thaumaturge_robe_feet                   |
+
+## 2. By removing the crafting recipe for a Dark Steel Upgrade
+
+**This is the better way to do it, since it has less chance of having unintended side effects.**
```

Comment:
The one downside is that upgrade items will still be generated as (and on) dungeon loot. Only disabling the upgrades as shown above (or disabling the dungeon loot completely) will prevent that.